MvvmHelpers/Commands/CommandUtils.cs

```diff
@@ -0,0 +1,40 @@
+﻿using MvvmHelpers.Exceptions;
+using System;
+using System.Reflection;
+
+namespace MvvmHelpers.Commands
+{
+	internal static class CommandUtils
+	{
+		internal static bool IsValidCommandParameter<T>(object o)
+		{
+			bool valid;
+			if (o != null)
+			{
+				// The parameter isn't null, so we don't have to worry whether null is a valid option
+				valid = o is T;
+
+				if (!valid)
+					throw new InvalidCommandParameterException(typeof(T), null);
+
+				return valid;
+			}
+
+			var t = typeof(T);
+
+			// The parameter is null. Is T Nullable?
+			if (Nullable.GetUnderlyingType(t) != null)
+			{
+				return true;
+			}
+
+			// Not a Nullable, if it's a value type then null is not valid
+			valid = !t.GetTypeInfo().IsValueType;
```

Comment:
Just FYI - `IsValueType` is only available in .NET Standard 2.0:  https://docs.microsoft.com/dotnet/api/system.type.isvaluetype.

It looks like MvvmHelpers' current minimum dependency is .NET Standard 1.0, so make sure to bump it up to .NET Standard 2.0.

MvvmHelpers also targets . NET 4.6.1, which is fine because `IsValueType` is supported in .NET v1.1.